gnome-audio-profiles-properties and related files are now provided by
media-libs/libgnome-media-profiles:3

So we don't build them, or install them to avoid file collisions

---
--- profiles/Makefile.am
+++ profiles/Makefile.am
@@ -1,4 +1,3 @@
-bin_PROGRAMS = gnome-audio-profiles-properties
 noinst_PROGRAMS =
 
 defines = \
@@ -21,11 +20,6 @@
 @INTLTOOL_SCHEMAS_RULE@
 
 uidir = $(pkgdatadir)
-ui_DATA = gnome-audio-profile-new.ui gnome-audio-profile-edit.ui
-
-gnome_audio_profiles_properties_SOURCES = gnome-audio-profiles-properties.c
-gnome_audio_profiles_properties_CFLAGS = $(GMP_CFLAGS) $(defines) -I$(top_srcdir)
-gnome_audio_profiles_properties_LDADD = libgnome-media-profiles.la $(GMP_LIBS)
 
 libgnome_media_profiles_la_SOURCES = \
 	audio-profile.c \
@@ -67,15 +61,8 @@
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gnome-media-profiles.pc
-EXTRA_DIST = $(ui_DATA) $(schema_DATA) $(pkgconfig_DATA)
+EXTRA_DIST = $(schema_DATA) $(pkgconfig_DATA)
 
 CLEANFILES = $(schema_DATA)
 
-# "." is dirty hack to first build our .la in this dir before moving to glade
-SUBDIRS =  . help
-
-if HAVE_GLADEUI
-SUBDIRS += glade
-endif
-
 -include $(top_srcdir)/git.mk
